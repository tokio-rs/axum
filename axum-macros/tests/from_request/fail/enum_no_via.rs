use axum_macros::FromRequest;

#[derive(FromRequest, Clone)]
enum Extractor {}

fn main() {}
