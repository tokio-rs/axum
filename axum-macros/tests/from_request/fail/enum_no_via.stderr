error: missing `#[from_request(via(...))]`
 --> tests/from_request/fail/enum_no_via.rs:3:10
  |
3 | #[derive(FromRequest, Clone)]
  |          ^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `FromRequest` (in Nightly builds, run with -Z macro-backtrace for more info)
